cmake出错了

/opt/aarch64/KPGCC/gcc-10.3.1-2023.12-aarch64-linux/bin/../lib64/gcc/aarch64-linux-gnu/10.3.1/../../../../aarch64-linux-gnu/bin/ld: ../lib/libexiv2.so.0.28.2: undefined reference to `ini_parse_string'
/opt/aarch64/KPGCC/gcc-10.3.1-2023.12-aarch64-linux/bin/../lib64/gcc/aarch64-linux-gnu/10.3.1/../../../../aarch64-linux-gnu/bin/ld: ../lib/libexiv2.so.0.28.2: undefined reference to `ini_parse'

cmake 3.25.0

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -280,10 +280,10 @@ if(Iconv_FOUND AND Iconv_LIBRARIES)
 endif()
 
 if(EXIV2_ENABLE_INIH)
-  target_link_libraries(exiv2lib_int PRIVATE inih::libinih)
   target_link_libraries(exiv2lib_int PRIVATE inih::inireader)
-  target_link_libraries(exiv2lib PRIVATE inih::libinih)
+  target_link_libraries(exiv2lib_int PRIVATE inih::libinih)
   target_link_libraries(exiv2lib PRIVATE inih::inireader)
+  target_link_libraries(exiv2lib PRIVATE inih::libinih)
   list(APPEND requires_private_list "INIReader")
 endif()
 

